addon/components/fabulous-table.js

```diff
@@ -69,9 +69,13 @@ export default Ember.Component.extend({
         let scrollContainerHeight = this.get('scrollContainer').outerHeight();
         let topOffsetOfScrollContainer = this.get('scrollContainer').offset().top;
         let bottomSelector = this.get('noSpinner') ? '.fabulous-row' : '.spinner';
-        let topOffsetOfLastItem = this.$(bottomSelector).last().offset().top;
-        
-        return topOffsetOfLastItem - topOffsetOfScrollContainer <= scrollContainerHeight;
+
+        if (this.$(bottomSelector).length > 0) {
```

Comment:
Fail fast: Inside this method check as soon as possible if there is no content and return `false`

Reply:
New comment below.